system-test/logging.ts

```diff
@@ -97,6 +98,20 @@ describe('Logging', () => {
       await getAndDelete(logging.getSinks.bind(logging));
     }
 
+    async function deleteLogs() {
+      const [logs] = await logging.getLogs();
```

Comment:
Using `getLogs()` and a naming convention consistent with the other test resources, we can clean up *all* test logs that are matched after each test run.